src/js/tracks/text-track-cue-list.js

```diff
@@ -57,14 +106,47 @@ TextTrackCueList.prototype.setCues_ = function(cues) {
       });
     }
   };
+  defineProp.call(this, this.cues_.length-1);
+  this.length_ = this.cues_.length;
+};
+
+TextTrackCueList.prototype.removeCue_ = function(removeCue) {
+  let removed = false;
+
+  for (let i = 0, l = this.cues_.length; i < l; i++) {
+    let cue = this.cues_[i];
+    if (cue === removeCue) {
+      this.cues_.splice(i, 1);
+      removed = true;
```

Comment:
should we break here? I don't think we're supposed to be able to have multiple of the same cue in the array.